examples/v1beta1/hp-tuning/grid-example.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: kubeflow.org/v1beta1
+kind: Experiment
+metadata:
+  namespace: kubeflow
```

Comment:
Although there is nothing wrong with defining the namespace, I could see some cases where this is not desired:
1) If users want to apply/create this outside the `kubeflow` namespace
2) If our test environment change and we do not deploy katib in the `kubeflow` namespace anymore

I'd suggest you leave this empty and apply/create the experiment specifying the namespace, which can be anything, like `testing`.

Reply:
would I then specify the namespace in the tox command?